cartoframes/data/services/geocoding.py

```diff
@@ -379,19 +381,23 @@ def _geocode(self, table_name, street, city=None, state=None, country=None, stat
 
         aborted = False
 
-        if not dry_run and self.available_quota() < output['required_quota']:
-            raise CartoException('You do not have enough quota. You need {} and you have {}'.format(
-                output['required_quota'],
-                self.available_quota()
-            ))
+        if not dry_run:
```

Comment:
Doing that, I would avoid checking `if not dry_run` in the following `if` (line 392)

Reply:
:+1: